rosdep/python.yaml

```diff
@@ -3465,6 +3469,10 @@ python3-pep8:
   ubuntu: [python3-pep8]
 python3-setuptools:
   ubuntu: [python3-setuptools]
+wheel:
```

Comment:
I changed the rule to use the packages you listed instead of pip.  Did I miss where these preferences are documented or would it be good to do so?